Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -190,4 +254,16 @@ void Processor::check_invoke_scheduler()
     }
 }
 
+NAKED void thread_context_first_enter(void)
+{
+    asm(
+        // FIXME: Implement this
```

Comment:
nit: put comment above `asm` line, then it formats nicer. Also, trailing `\n` not needed i think (?)

Reply:
The trailing `\n` is actually needed as the string literals are just concatenated, and send to the assembler. And the assembler doesn't like it if there are no newlines.